[DOCS] Host and user risk score alert enrichments (#2558)
elastic/security-docs: docs/detections/alerts-view-details.asciidoc

* *Insights*: Shows relationships with associated alerts to help you quickly identify patterns. Refer to <<alert-details-insights, Insights on alerts>> for more information.

> * *Enriched data*: Displays risk scores for users and hosts and available threat intelligence data. Refer to <<alerts-enrich-host-user-risk-score>> and <<alerts-enrich-ti>> to learn more.

Reviewer: ```suggestion
* *Enriched data*: Displays risk scores for users and hosts, as well as threat intelligence data. Refer to <<alerts-enrich-host-user-risk-score>> and <<alerts-enrich-ti>> to learn more.
```

Author: Hm, I'm not sure if the suggested edit is correct. From my perspective, it indicates that threat intelligence is always offered for all alerts -- though feel free to correct me if I'm misinterpreting the edit. My understanding is that it only shows up if it's available in the alert's document. @nkhristinin am I relaying the correct information?

Reviewer 2: Maybe combine the two, "risk scores for users and hosts, as well as available threat intel"? I think @benironside's edit is trying to smooth out the list of "x and y and z", so "as well as" works as long as it doesn't lose "available"?

Author: Gotcha, and this works^

Author: ```suggestion
* *Enriched data*: Displays risk scores for users and hosts, as well as available threat intelligence. Refer to <<alerts-enrich-host-user-risk-score>> and <<alerts-enrich-ti>> to learn more.
```

* *Alerts related by process ancestry* - Shows alerts that are related by process events on the same linear branch. Note that alerts generated from processes on child or related branches are not shown. To further examine alerts, click *Investigate in timeline*.

[discrete]
[[alerts-enrich-host-user-risk-score]]
==== Alerts enriched with user and host risk scores
Alerts can be enriched with user and host risk scores, which convey the level of risk associated with a specific user and host. Risk levels can be `Unknown`, `Low`, `Moderate`, `High`, or `Critical`. Refer to <<host-risk-score>> and <<user-risk-score>> to learn how risk scores are calculated.

> IMPORTANT: User and host risk scores are technical preview features. You must enable user and host risk score features to access risk scores data. Refer to [anchor] to learn more.

Reviewer 3: Anchor not a link
<img width="818" alt="Screenshot 2022-10-10 at 14 10 02" src="https://user-images.githubusercontent.com/7609147/194863354-0b9e3bed-d18d-4727-9674-327af4999293.png">

Author: This is actually intentional. I've been leaving it as is since it doesn't point to anything yet and will break the doc build if I add it.

[role="screenshot"]
image::images/enriched-host-user-rs.png[Host and user risk score subsections with risk scores, 600]

Two types of user and host risk scores can be shown in the Enriched data section:

* **Current user/host risk classification:** The current risk score of the user or host that is associated with the alert.

> * **Original user/host risk classification:** The first risk score that was calculated for the user or host that is associated with the alert. The *Original user/host risk classification* field always accompanies the *Current user/host risk classification* field to show how the user or host's risk score has changed over time.

Reviewer 3: For the original risk score, should we mention that we show it only if it is different form the current?

Author: I can be more explicit about this. The behavior is loosely implied in the last sentence so I can modify that.

Author: ```suggestion
* **Original user/host risk classification:** The first risk score that was calculated for the user or host that is associated with the alert. The *Original user/host risk classification* field only displays if the current risk score no longer matches the original risk score. In this situation, both the current and original risk scores are displayed to show how they changed over time. The current risk score is provided in the *Current user/host risk classification* field. The original risk score is provided in the *Original user/host risk classification* field. 
```

* *Alerts related by process ancestry* - Shows alerts that are related by process events on the same linear branch. Note that alerts generated from processes on child or related branches are not shown. To further examine alerts, click *Investigate in timeline*.

[discrete]
[[alerts-enrich-host-user-risk-score]]
==== Alerts enriched with user and host risk scores

> Alerts can be enriched with user and host risk scores, which convey the level of risk associated with a specific user and host. Risk levels can be `Unknown`, `Low`, `Moderate`, `High`, or `Critical`. Refer to <<host-risk-score>> and <<user-risk-score>> to learn how risk scores are calculated.

Reviewer 2: Revise for repetition of "can be"
```suggestion
Alerts can be enriched with user and host risk scores, which convey the level of risk associated with a specific user and host. Risk levels are `Unknown`, `Low`, `Moderate`, `High`, or `Critical`. Refer to <<host-risk-score>> and <<user-risk-score>> to learn how risk scores are calculated.
```

> IMPORTANT: User and host risk scores are technical preview features. You must enable user and host risk score features to access risk scores data. Refer to [anchor] to learn more.

Reviewer: ```suggestion
IMPORTANT: User and host risk scores are in technical preview. You must manually enable them to access risk score data. Refer to [anchor] to learn more.
```

> NOTE: User and host risk scores are technical preview features. You must enable user and host risk score features to access risk scores data. Refer to <<host-risk-score>> and <<user-risk-score>> to learn more.

Reviewer 4: ```suggestion
NOTE: User and host risk scores are technical preview features and require a www.elastic.co/pricing[Platinum subscription] or higher. You must enable user and host risk score features to access risk scores data. Refer to <<host-risk-score>> and <<user-risk-score>> to learn more.
```

> NOTE: User and host risk scores are technical preview features. You must enable user and host risk score features to access risk scores data. Refer to <<host-risk-score>> and <<user-risk-score>> to learn more.

Author: Note to self: Might need to update these anchors one more time after host and user risk score install docs are updated. ATM, an anchor exists for the user risk score install docs, but one isn't available for the host risk score install docs.

Reviewer 2: Would updating these anchors break any existing URLs? I don't know the architecture exactly, but I'm a little paranoid about renaming anchors after I burned myself a few releases back when I broke some URLs and had to request some redirects for the old -> new URLs.

Author: I'm not actually changing the anchor names. I left this note for myself in case an anchor was added to [this section](https://www.elastic.co/guide/en/security/master/host-risk-score.html#_deploy_host_risk_score) of the host risk score docs. Since the section doesn't have anchor atm, I'm pointing to the [main topic](https://www.elastic.co/guide/en/security/master/host-risk-score.html). 

I might actually resolve this comment early though. The install instructions for both risk score features are fairly easy to find in the linked topics, so adding new anchor links to my topic aren't all that necessary.

Reviewer 2: Cool, thanks for explaining!

[role="screenshot"]
image::images/enriched-host-user-rs.png[Host and user risk score subsections with risk scores, 600]

> Two types of user and host risk scores can be shown in the Enriched data section:

Reviewer: ```suggestion
Two types of user and host risk scores can appear in the Enriched data section:
```

> * **Current user/host risk classification:** The current risk score of the user or host that is associated with the alert.

Reviewer: ```suggestion
* **Current user/host risk classification:** The current risk score of the user or host associated with the alert.
```

> * **Original user/host risk classification:** The first risk score that was calculated for the user or host that is associated with the alert.

Reviewer: ```suggestion
* **Original user/host risk classification:** The first risk score that was calculated for the user or host associated with the alert.
```

+

> The *Original user/host risk classification* field only displays if the current risk score no longer matches the original risk score. In this situation, both the current and original risk scores appear, showing how the risk changed. The current risk score appears in the *Current user/host risk classification* field. The original risk score appears in the *Original user/host risk classification* field.

Reviewer 2: The last two sentences seem pretty obvious, based on the field names. We also just explained this in the bullets above
```suggestion
The *Original user/host risk classification* field only displays if the current risk score no longer matches the original risk score. In this situation, both the current and original risk scores appear, showing how the risk changed.
```

> The *Original user/host risk classification* field only displays if the current risk score no longer matches the original risk score. In this situation, both the current and original risk scores are displayed to show how they changed over time. The current risk score is provided in the *Current user/host risk classification* field. The original risk score is provided in the *Original user/host risk classification* field.

Reviewer: ```suggestion
The *Original user/host risk classification* field only displays if the current risk score no longer matches the original risk score. In this situation, both the current and original risk scores appear, showing how the risk changed. The current risk score appears in the *Current user/host risk classification* field. The original risk score appears in the *Original user/host risk classification* field.
```

[discrete]
[[alerts-enrich-ti]]
==== Alerts enriched with threat intelligence

> Alerts can be enriched with contextually relevant threat intelligence that you can use for triaging and investigating alerts. If available, this information appears under the Enriched data section.

Reviewer 2: Nit: "in" is more flexible than "under" if the UI ever changes, and is somewhat more accessible too.
```suggestion
Alerts can be enriched with contextually relevant threat intelligence that you can use for triaging and investigating alerts. If available, this information appears in the Enriched data section.
```

> Alerts can be enriched with contextually relevant threat intelligence that you can use for triaging and investigating alerts. If available, this information is displayed under the Enriched data section.

Reviewer: ```suggestion
Alerts can be enriched with contextually relevant threat intelligence that you can use for triaging and investigating alerts. If available, this information appears under the Enriched data section.
```

If these fields aren't available, {elastic-sec} does not perform the query and does not display threat intelligence data for the alert.

> Available threat indicator match data is placed under the **Threat Match Detected** subsection of the Enriched data section.

Reviewer 2: Prob not necessary to specify all the levels and sublevels, we've already established that we're talking about the main "Enriched data" section
```suggestion
Available threat indicator match data appears in the **Threat Match Detected** subsection.
```

> Discovered threat indicator match data is placed under the **Threat Match Detected** subsection of the Enriched data section.

Reviewer: ```suggestion
Discovered threat indicator match data appears under the **Threat Match Detected** subsection of the Enriched data section.
```
I'm not totally clear on what "Discovered" means here. Would "Relevant" or "Available" maybe be a better fit?

Author: ```suggestion
Available threat indicator match data is placed under the **Threat Match Detected** subsection of the Enriched data section.
```

[role="screenshot"]
image::images/matched-indicator-sub-sec.png[Threat Match Detected subsection with matched file hash, 600]

> All other available threat intelligence data appears under the **Enriched with Threat Intelligence** subsection.

Reviewer 2: ```suggestion
All other available threat intelligence data appears in the **Enriched with Threat Intelligence** subsection.
```

> All other available threat intelligence data is placed under the **Enriched with Threat Intelligence** subsection.

Reviewer: ```suggestion
All other available threat intelligence data appears under the **Enriched with Threat Intelligence** subsection.
```
